test/tests/functional/pbs_hook_management.py

```diff
@@ -318,6 +318,95 @@ def test_hook_02(self):
 
         self.logger.info("**************** HOOK END ****************")
 
+    def test_hook_03(self):
+        """
+        By creating an import hook, it executes a management hook.
+        Also sets debug to True.
+        """
+        self.logger.info("**************** HOOK START ****************")
+        hook_name_a = "management_03a"
+        hook_msg_a = 'running management hook_03a'
+        hook_body_a = get_hook_body(hook_msg_a)
+        attrs = {'event': 'management', 'enabled': 'True', 'debug': 'True'}
+        start_time_a = time.time()
+        ret = self.server.create_hook(hook_name_a, attrs)
+        self.assertEqual(ret, True, f"Could not import hook {hook_name_a}")
```

Comment:
Update the error message "Could not create hook"

Reply:
Updated